server/src/main/java/org/elasticsearch/index/query/MatchPhraseQueryBuilder.java

```diff
@@ -75,6 +80,7 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeString(fieldName);
         out.writeGenericValue(value);
         out.writeVInt(slop);
+        zeroTermsQuery.writeTo(out);
```

Comment:
Same here, write only the `out.getVersion` is before 7.

Reply:
Guessing you mean 'after 7' here.

Reply:
Yep, sorry